frontend/src/app/patients/EditPatient.tsx

```diff
@@ -313,20 +321,27 @@ const EditPatient = (props: Props) => {
         </div>
       </div>
       <div className="display-flex flex-align-center">
-        {!props.fromQueue && (
+        {!props.fromQueue && formChanged ? (
+          <Button
+            id="edit-patient-save-upper"
+            className="prime-save-patient-changes-start-test"
+            disabled={loading}
+            onClick={() => {
+              onSave(true);
+            }}
+            variant="outline"
+            label={"Save and start test"}
+          />
+        ) : (
           <Button
             id="edit-patient-save-upper"
             className="prime-save-patient-changes-start-test"
-            disabled={loading || !formChanged}
```

Comment:
⚠️ The logic for `formChanged` is moved up to the turnery on line 324 in the new code.